functions/openhab.bash

```diff
@@ -6,10 +6,9 @@
 ##    create_sys_dependencies()
 ##
 create_sys_dependencies() {
-  local targetDir
-
-  targetDir="/etc/systemd/system/openhab2.service.d"
+  local targetDir="/etc/systemd/system/openhab2.service.d"
```

Comment:
Declare and assign on different lines.

Reply:
yes, that's where I disagreed with your code style. I don't mind either method.

Reply:
I prefer it because it reduces ambiguity, and at some point I ran into issues with ShellCheck if I didn't declare and assign separately.

Reply:
Both ways are fine. Use what you think is best depending on situation.